examples/mobile/UIComponents/components/progress/progress-bar/progressbar.css

```diff
@@ -0,0 +1,22 @@
+.progress-container {
```

Comment:
I'm not sure what part of these styles should belong to framework and which to the app.
What do you think about create class `.ui-progress-container` and `.ui-progress-text` ?